packages/cyberstorm/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "@thunderstore/cyberstorm",
+  "version": "0.1.0",
+  "description": "Shared components for Thunderstore",
+  "repository": "https://github.com/thunderstore-io/thunderstore-ui/tree/master/packages/cyberstorm",
+  "main": "dist/thunderstore-cyberstorm.cjs.js",
+  "module": "dist/thunderstore-cyberstorm.esm.js",
+  "types": "dist/thunderstore-cyberstorm.cjs.d.ts",
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "build": "tsc",
+    "dev": "tsc --watch"
+  },
+  "dependencies": {
+    "@babel/runtime": "^7.17.0",
+    "@emotion/react": "^11.4.1",
```

Comment:
IIRC `@emotion` is a dependency for Chakra UI. Same goes for `framer-motion`. Some others, such as `downshift` and `s-ago` are used by the old components, but not really needed here. Please clean the list to only include the actual dependencies.

Reply:
removed some unused packages, did I miss some?

Reply:
TBH quite impossible to remember/tell by looking at the file. But since `cyberstorm` currently contains only the button element, I'd say the following are propably not needed: `@thunderstore/hooks`, `js-cookie`, `react-dropzone`, `react-hook-form`, `react-markdown`, `react-query`, `remark-gfm`, `@types/js-cookie`.

Reply:
Removed those also.